source/2021-11-06-the-ember-times-issue-167.md

```diff
@@ -24,14 +24,15 @@ Read more about the new changes to [ember-a11y-testing on GitHub](https://github
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Release 2.0.x of ember-select-light 🎉](https://github.com/ember-a11y/ember-select-light/releases/tag/v2.0.0)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Ava Wroten (@hergaiety)](https://www.wroten.me/) released 2.0.0 of [ember-select-light](https://github.com/ember-a11y/ember-select-light/) moving the addon into the [ember-a11y org](https://github.com/ember-a11y) (join us on [discord in #dev-ember-a11y](https://discord.gg/xBzFBMYv)) and making it **Octane ready**.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+**ember-select-light is a simple and accessible `<select>` dropdown component** ✨. With a simple API you can pass in a flat or nested array of options or yield out to your own manually handled options template code. This component pairs nicely with the latest [ember-component-patterns for Select Elements](https://emberjs-1.gitbook.io/ember-component-patterns/form-components/select-element).
+
+With the new 2.0.0 drops support for older versions of Node and Ember. Node v10 LTS and Ember.js v3.16 LTS are suggested. Some breaking syntax changes (see the [upgrade guide](https://github.com/ember-a11y/ember-select-light/blob/main/UPGRADE.md)) Lastly, documentation and plenty of housekeeping was done leading up to this release.
```

Comment:
I wasn't sure what you meant by housekeeping. I think we could remove the part of the sentence for clarity?

```suggestion
The 2.0.0 release drops support for older versions of Node and Ember. At minimum, Node v10 LTS and Ember.js v3.16 LTS are suggested. It also introduces some breaking syntax changes (see the [upgrade guide](https://github.com/ember-a11y/ember-select-light/blob/main/UPGRADE.md)). Lastly, documentation has been updated.
```